IoTDeviceSimulator/web-client/src/App.css

```diff
@@ -134,12 +134,18 @@ header button {
   max-width: 430px !important;
   min-width: 430px !important;
 }
-
-.table-top {
+.table-top-wrap{
   display: flex;
   flex-direction: row;
+  justify-content: space-between;
+}
+.table-top-left {
+  display: flex;
```

Comment:
Are these styles (table-top-right, table-top-left) getting used somewhere?, if not it can be removed